File: app/src/main/java/com/movieapp/huxymovies/model/MovieApiResponse.java

package com.movieapp.huxymovies.model;

import java.util.List;
import com.google.gson.annotations.SerializedName;

@SuppressWarnings("unused")
public class MovieApiResponse {

    @SerializedName("page")
    public Long mPage;
    @SerializedName("results")
    public List<Result> mResults;
    @SerializedName("total_pages")
    public Long mTotalPages;
    @SerializedName("total_results")
    public Long mTotalResults;

}

File: app/src/main/java/com/movieapp/huxymovies/model/MovieApiResponse.kt
package com.movieapp.huxymovies.model

import com.google.gson.annotations.SerializedName

class MovieApiResponse {

    var id: Int? = null
    @SerializedName("page")
    var mPage: Long? = null
    @SerializedName("results")
    var mResults: List<Result>? = null
    @SerializedName("total_pages")
    var mTotalPages: Long? = null

}
